google-cloud-translate/.rubocop.yml

```diff
@@ -4,12 +4,22 @@ inherit_gem:
 AllCops:
   Exclude:
     - "acceptance/**/*"
-    - "google-cloud-translate.gemspec"
-    - "Rakefile"
+    # - "google-cloud-translate.gemspec"
+    - "lib/google/cloud/translate/v3.rb"
+    - "lib/google/cloud/translate/v3/**/*"
+    # - "Rakefile"
```

Comment:
Drop these commented out lines if not needed.

Reply:
Oops.